client/src/reducers/index.js

```diff
@@ -26,3 +27,12 @@ export default combineReducers({
   achievements: achievementsReducer, 
   homepage: homepageReducer,
 });
+
+const rootReducer = (state, action) => {
+  if (action.type === LOG_OUT) {
+    return appReducer(undefined, action)
+  }
+  return appReducer(state, action)
+}
+
+export default rootReducer
```

Comment:
Could you document this somewhere with the link to the stack overflow post to make clear what we're doing here? Perhaps add a line in the docstring at the start of this file.